specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/azureFirewall.json

```diff
@@ -272,6 +272,9 @@
           }
         ],
         "responses": {
+          "202": {
```

Comment:
>"202": { [](start = 10, length = 8)

2020-03-01 is already public.  Supporting async after the fact is a breaking change for existing clients.  Please make this change in a new api-version.

Reply:
Also, why does PATCH need to be async?  The operation description says it is only updating tags.

Reply:
2020-03-01 is not in public for network RP.

We need it to be async call so that the tags are sent to the billing pipeline.

Reply:
It is in your public manifest which means it is public:  https://armmanifest.visualstudio.com/One/_git/Manifest?path=%2FMICROSOFT.NETWORK%2FPROD%2FMANIFEST.JSON&version=GBmaster&line=89571&lineEnd=89571&lineStartColumn=13&lineEndColumn=25&lineStyle=plain